cmd/gpu-kubelet-plugin/driver.go

```diff
@@ -177,6 +202,68 @@ func (d *driver) nodeUnprepareResource(ctx context.Context, claimNs kubeletplugi
 	return nil
 }
 
+func (d *driver) deviceHealthEvents(ctx context.Context, nodeName string) {
+	klog.Info("Processing device health notifications")
+	for {
+		select {
+		case <-ctx.Done():
+			klog.V(6).Info("Stop processing device health notifications")
+			return
+		case device, ok := <-d.deviceHealthMonitor.Unhealthy():
+			if !ok {
+				klog.V(6).Info("Health monitor channel closed")
+				return
+			}
+			uuid := device.UUID()
```

Comment:
Can `device.UUID()` fail? Edit: I see, it doesn't -- cf `func (d AllocatableDevice) UUID() string`.